### ISSUE TITLE
"Only Show Unsigned Chat"

### ISSUE THREAD

**Issue by IlyaTheSawmill** [Aizistral-Studios/No-Chat-Reports#417]
### Idea

Do the opposite of the vanilla Minecraft - change the "Only Show Secure Chat" option to "Only Show Unsigned Chat".

### Reasoning

Just make players who has No Chat Reports cannot see signed messages.
It is somewhat similar to #373

### Other Information

Disabled by default

**Comment by Madis0** [comment]
Where would anyone _benefit_ from this setting? 

There are quite few servers that even allow raw unsigned messages (compared to disallowing or converting them), mostly vanilla ones. So, on those servers - that have few players anyway - you'd just not see most users' messages? Maybe there'd be some angry admins telling you to enable chat reporting and you'd never see it.

I'd understand the viewpoint if raw unsigned messages would be allowed on big servers like Hypixel, so you could have like a "group chat" of sorts, but since it isn't, I feel like this option would just break the game even more than the original one.

**Comment by IlyaTheSawmill** [comment]
> There are quite few servers that even allow raw unsigned messages

The vast majority of servers (at least those I have played) have enforce-secure-profile=false set, and I rarely see signed messages on them.

The "group chat" is one of my idea.

**Comment by Madis0** [comment]
> The vast majority of servers (at least those I have played) have enforce-secure-profile=false set, and I rarely see signed messages on them.

That is true, but in the majority you don't see _unsigned messages_ either, you just see the system messages, hence my question.

**Comment by IlyaTheSawmill** [comment]
However, system messages are not signed messages.

Players mostly use unoffical launchers, they cannot fetch keys. This means they CANNOT send signed messages at all (with or without NCR).

So I just want a function like this :) If you won't add it, I won't be sad

**Comment by Madis0** [comment]
> Players mostly use unoffical launchers, they cannot fetch keys. This means they CANNOT send signed messages at all (with or without NCR).

In such servers, it is not possible to send signed messages even with an official launcher. Hence I'm trying to find use cases where this would benefit the user. 

So, do you know any servers where:

- chat signing is optional
- chat is _not_ converted to system messages 
- at least 50% of users are aware of NCR?

**Comment by IlyaTheSawmill** [comment]
mc.kianakiferi.cn and mc.touchafish.cc for 1.20.1

**Comment by Madis0** [comment]
> mc.kianakiferi.cn and mc.touchafish.cc for 1.20.1

Hmm. So I tested them out:
* The first one keeps complaining that authentication servers are down, even though I can join every other online server just fine.
* The second one kicked me with the reason "please use a genuine account to join the server", though after a relog it still let me in (online account). Though yes, you're right about the chat signing being optional in this server.

With that in mind, I am not sure about your claim

> Players mostly use unoffical launchers, they cannot fetch keys. This means they CANNOT send signed messages at all (with or without NCR).

if neither of the servers didn't even want to let me in due to authentication problems...

**Comment by IlyaTheSawmill** [comment]
> Players mostly use unoffical launchers, they cannot fetch keys. This means they CANNOT send signed messages at all (with or without NCR).
This's the TRUE situation in China.
An example: https://github.com/Hex-Dragon/PCL2
